src/olympia/addons/cron.py

```diff
@@ -180,29 +179,22 @@ def deliver_hotness():
     threshold = 250 if addon type is theme, else 1000
     hotness = (a-b) / b if a > threshold and b > 1 else 0
     """
-    frozen = set(f.addon_id for f in FrozenAddon.objects.all())
-    averages = get_averages_by_addon_from_bigquery(today=date.today())
-    addons = (
-        Addon.objects.filter(guid__in=averages.keys())
-        .filter(status__in=amo.REVIEWED_STATUSES)
-        .exclude(id__in=frozen)
+    frozen_guids = list(set(fa.addon.guid for fa in FrozenAddon.objects.all()))
```

Comment:
we have to wrap this in a `set`?  That's a bit odd.  (Even if there were duplicates, does the break something?  If so, it should probably be in `get_averages_by_addon_from_bigquery`)

Reply:
🤷‍♂️ meh though